zk-token-sdk/src/zk_token_proof_instruction.rs

```diff
@@ -130,12 +142,10 @@ pub enum ProofInstruction {
     ///
     VerifyTransferWithFee,
 
-    /// Verify a pubkey validity zero-knowledge proof.
+    /// Verify a public-key validity zero-knowledge proof.
```

Comment:
Seems like we've already committed to using the "pubkey" term for ElGamal public keys, so I'm inclined to leave this.

Reply:
Yeah, unfortunately, I used "pubkey" and "public key" (and even "public-key" with a hyphen :pensive:) interchangeably in `sigma_proofs` module as well. In fact, this seems to be the case in other parts of the monorepo as well. In `pubkey.rs` in the `sdk`, both "public key" and "pubkey" are used in the docs.

I think "pubkey" is a Solana-specific term for structs and "public key" is academically the proper way to refer to things, so let's change this to "public key" (without a hyphen).

Reply:
I actually didn't mind `pubkey` here, since the instruction is called `VerifyPubkeyValidity`. But your edited version is okay.
To be clear, you're thinking `pubkey` for structs, variables, and code generally, but it ought to be `public key` when talking about those things?

Reply:
Yes that was how I was thinking about it, but I am open to suggestions if you have any strong opinions on it. It seems like this is how it is generally done in the`solana-sdk` though there are some bits that uses "pubkey" in the docs descriptions.